src/dist/replication/meta_server/meta_bulk_load_service.cpp

```diff
@@ -20,14 +20,13 @@ bulk_load_service::bulk_load_service(meta_service *meta_svc, const std::string &
 void bulk_load_service::initialize_bulk_load_service()
 {
     task_tracker tracker;
-    error_code err = ERR_OK;
+    _sync_bulk_load_storage =
```

Comment:
Is there any other initialization point of `_sync_bulk_load_storage`? Given that It's inited with a temporary tracker here, can you assure it's memory-safe for `_sync_bulk_load_storage` to be called in other functions?